### PR TITLE
(Rebased) Update emergency-start semantics for chronos jobs

### PR DESCRIPTION
**Pull request by nhandler**
Since I am unable to push changes to @keshavdv's fork, I had to create a new branch and Pull Request. This is the same as #268, but I rebased on top of master.

### CODE REVIEW

File: paasta_tools/chronos_tools.py [hunk 1]
```diff
@@ -597,6 +596,27 @@ def get_key(job):
...
+    node,example_job,success,queued
+    node,parent_job,success,idle
+    node,child_job,success,idle
+    link,parent_job,child_job
+
+    :param client: Chronos client object
+    :param service: service name
+    :param instance: instance name
+    """
+    resp = client._call("/scheduler/graph/csv", "GET")
```
Comment (mjksmith) [inline paasta_tools/chronos_tools.py]: This can now be replaced with `client.scheduler_graph()`

**Comment by keshavdv** [comment]
During testing I noticed that starting a manual run of a disabled job doesn't actually trigger an execution anymore. I'm not sure if this behavior changed recently in Chronos, but it might mean we have to rethink the semantics or the implementation. Or maybe I just have some wrong config for the service that makes this not work?

Logs:
```
[2016-05-31 17:50:03,089] INFO Updating state for job (example_happyhour example_chronos_job) to queued (org.apache.mesos.chronos.scheduler.jobs.stats.JobStats:62)
... (a little while later)
[2016-05-31 17:50:04,297] INFO JobNotificationObserver does not handle JobExpired(ScheduleBasedJob(R/2016-04-15T06:00:00Z/PT24H,example_happyhour example_chronos_job,/code/virtualenv_run/bin/python -m timeit,PT60S,0,0,,,2,noop,,configab5fb2d9,...)
[2016-05-31 17:50:04,298] INFO Updating state for job (example_happyhour example_chronos_job) to idle (org.apache.mesos.chronos.scheduler.jobs.stats.JobStats:62)
```

**Comment by keshavdv** [comment]
Based on Chronos's behavior, the final semantics of this PR are:

emergency-start with previously stopped jobs: noop (warn user)
emergency-start with 'disabled: False' in config: Trigger manual run immediately
emergency-start with 'disabled: True' in config: noop (warn user)
emergency-stop: Stop in flight job and mark as disabled in Chronos

File: paasta_itests/steps/chronos_steps.py [hunk 2]
```diff
@@ -67,15 +69,22 @@ def list_chronos_jobs_has_job(context, should_or_not):
...
-# NOTE this is a placeholder until we are able to get per-job task information from Chronos
-@then(u"the {old_or_new_job} {has_or_not} running tasks")
-def chronos_check_running_tasks(context, old_or_new_job, has_or_not):
-    # job_id = context.old_chronos_job_name if old_or_new_job == 'old job' else context.chronos_job_name
-    # if has_or_not == "has no":
-    #     assert job_id has no running tasks
-    # else:  # has_or_not should be "has"
-    #     assert job_id has running tasks
-    assert True
+@then(u'the job stored as "{job_name}" {has_or_not} running tasks')
```
Comment (solarkennedy) [inline paasta_itests/steps/chronos_steps.py]: What if this was just 

```
"the {job_name} job is {status}"
```

And then you can just assert that "myjob" is "idle", etc?

**Comment by solarkennedy** [comment]
I say ship

**Pull request merged**